controllers/gatekeeper_controller.go

```diff
@@ -89,9 +90,12 @@ var (
 		AuditFile,
 		WebhookFile,
 		"v1_service_gatekeeper-webhook-service.yaml",
+	}
+	staticWebhookAssets = []string{
```

Comment:
nit: s/staticWebhook/webhookStatic/ for consistency with the other variable names.

Reply:
fixed